src/pages/host/getHostEventString.tsx

```diff
@@ -212,7 +211,7 @@ export const getHostEventString = (
           Assigned to run task{" "}
           <StyledLink
             data-cy="host-running-task-set-link"
-            href={`${getUiUrl()}/task${data.taskId}/${data.execution}`}
+            href={`/task/${data.taskId}/${data.execution}`}
```

Comment:
Also this url pattern isn't correct for tasks.
Task urls follow the following pattern
`/task/:id/:tab?url_params`
url_params being `execution` in this case